ouija.js

```diff
@@ -316,6 +316,8 @@ function updatePostFlair(post, response){
 //awesome workaround from https://mathiasbynens.be/notes/javascript-unicode
 //for getting accurate character count even when handling emojis
 function countSymbols(string) {
+	// Do not count emoji variations and modifiers, http://unicode.org/reports/tr51/index.html
+	string = string.replace(/[\u{20E0}\u{20E3}\u{FE0E}\u{FE0F}\u{1F3FB}-\u{1F3FF}]/ug,'');
```

Comment:
Oh I see, this would potentially allow more emojis from being weeded out. Are you certain that these emojis would cause countSymbols to return > 1?